code/stable/gamephysics/SRS/HTML/GamePhysics_SRS.html

```diff
@@ -952,7 +952,7 @@ <h3>Theoretical Models</h3>
                     <tr>
                       <th>Equation</th>
                       <td>
-                        \[\symbf{F}=G\,\frac{{m_{1}}\,{m_{2}}}{\|\symbf{d}\|^{2}}\,\symbf{\hat{d}}=G\,\frac{{m_{1}}\,{m_{2}}}{\|\symbf{d}\|^{2}}\,\frac{\symbf{d}}{\|\symbf{d}\|}\]
+                        \[\symbf{F}=G\,\frac{m_{1}\,m_{2}}{\|\symbf{d}\|^{2}}\,\symbf{\hat{d}}=G\,\frac{m_{1}\,m_{2}}{\|\symbf{d}\|^{2}}\,\frac{\symbf{d}}{\|\symbf{d}\|}\]
```

Comment:
This is one of the cases where the old one might have been better? `\,{m_{2}}` might be better latex than `\,m_{1}` ?

Reply:
I'm fairly sure that we introduced those extra curly braces to fix a rendering bug. The fix was probably overly brutal, and I agree that fewer of them is a real improvement. Maybe try to use `git blame` to figure out the history?